components/LikeRegisterForm.vue

```diff
@@ -279,8 +292,30 @@ export default {
         }
       }
     },
+    async fetchReferrerInfo() {
+      this.shouldShowReferrerDialog = true;
+      this.setErrorDisabled(true);
+      try {
+        const response = await api.apiGetUserMinById(this.referrer);
```

Comment:
probably want to use a action that call 
```
const { avatar, displayName } = await apiWrapper(ctx, api.apiGetUserMinById(r.id), { slient: true });
```
this kind of style is when apiWrapper didnt support `slient`

Reply:
Updated